sys-apps/util-linux/util-linux-2.39.ebuild

```diff
@@ -173,6 +173,7 @@ python_configure() {
 		--enable-libblkid
 		--enable-libmount
 		--enable-pylibmount
+		--runstatedir=/run
```

Comment:
This should probably be `--runstatedir="${EPREFIX}/run"` instead.

Reply:
Done.